conn.go

```diff
@@ -467,7 +467,7 @@ func (c *Conn) handleRcpt(arg string) {
 	recipient := strings.Trim(arg[3:], "<> ")
 
 	if c.server.MaxRecipients > 0 && len(c.recipients) >= c.server.MaxRecipients {
-		c.writeResponse(552, EnhancedCode{5, 5, 3}, fmt.Sprintf("Maximum limit of %v recipients reached", c.server.MaxRecipients))
+		c.writeResponse(452, EnhancedCode{4, 5, 3}, fmt.Sprintf("Maximum limit of %v recipients reached", c.server.MaxRecipients))
```

Comment:
It seems like the only valid short code for x.5.3 is 451, according to https://www.iana.org/assignments/smtp-enhanced-status-codes/smtp-enhanced-status-codes.xhtml

Reply:
I found that [RCF 5321](https://www.rfc-editor.org/rfc/rfc5321#section-4.5.3.1.9) states that basic status code for "Too many recipients" is `452`.  The "Associated Basic Status Code" field in IANA registry does not cover all possible values,

Reply:
Right!